app/workers/metrics/record_daily_notifications_worker.rb

```diff
@@ -0,0 +1,34 @@
+module Metrics
+  class RecordDailyNotificationsWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :low_priority, retry: 10
+
+    EVENT_TITLES = %w[
+      welcome_notification_set_up_profile
+      welcome_notification_welcome_thread
+      welcome_notification_customize_feed
+      welcome_notification_twitter_connect
+      welcome_notification_github_connect
+      welcome_notification_customize_experience
+      welcome_notification_discuss_and_ask
+      welcome_notification_download_app
+      welcome_notification_ask_question
+      welcome_notification_start_discussion
+    ].freeze
+
+    def perform
+      # Welcome Notification click events created in the past day, logged by title.
+      EVENT_TITLES.each do |title|
+        event = Ahoy::Event.where(name: "Clicked Welcome Notification").
+          where("time > ?", 1.day.ago).
+          where("properties->>'title' = ?", title)
+
+        DatadogStatsClient.count(
+          "ahoy_events.#{title}",
+          event.size,
+          resource: "ahoy_events",
```

Comment:
I'm not sure if this is the "right" value for `resource`? I had trouble finding advice around this in the [documentation](https://docs.datadoghq.com/developers/metrics/dogstatsd_metrics_submission/?tab=ruby) but maybe I just missed something. Let me know if you have guidance around how I should be thinking about this parameter :)

Reply:
@mstruve LOL i got `resource` from our docs I think: https://docs.dev.to/backend/metrics/#time-series-data

Reply:
@mstruve Ah, I like the idea you're suggesting. Something more like this?
```ruby
DatadogStatsClient.count(
          "ahoy_events",
          event.size,
          tags: ["title:#{title}"],
)
```